modules/installation/proc-install-rhdh-orchestrator-helm.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies
+// assembly-orchestrator-helm-modules.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-install-orchestrator-helm_{context}"]
+= Install Orchestrator Helm
+
+You can install the Orchestrator flavor of {product} ({product-very-short}) on OpenShift using Helm charts, enabling serverless workflows and workflow orchestration capabilities.
+
+.Prerequisites
+
+* You are logged in as an administrator on the OpenShift cluster.
+* You have installed Helm and configured locally.
+* You have access to the {product} Helm chart repository.
+
+.Procedure
+
+. Add required Helm repositories:
++
+[source,yaml]
+----
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo add backstage https://backstage.github.io/charts
+helm repo add redhat-developer https://redhat-developer.github.io/rhdh-chart
+----
+
+. Install the orchestrator infrastructure Helm chart:
++
+[source,yaml]
+----
+helm install <release_name> redhat-developer/redhat-developer-hub-orchestrator-infra
+----
+
+. Manually approve the Install Plans for OpenShift Serverless and Serverless Logic Operators and wait for deployment.
+
+. Install the Backstage chart with the orchestrator enabled:
++
+[source,bash]
+----
+helm install <release_name> redhat-developer/backstage --set orchestrator.enabled=true
+----
+
+. To disable serverlessLogicOperator and serverlessOperator if managed externally:
++
+[source,bash]
+
+. (Optional) Enable Notifications and Signals plugins by adding the following to your values.yaml before installation or upgrade:
++
+[source,yaml]
+----
+disabled: false
+package: "./dynamic-plugins/dist/backstage-plugin-notifications"
+
+disabled: false
+package: "./dynamic-plugins/dist/backstage-plugin-signals"
+
+disabled: false
+package: "./dynamic-plugins/dist/backstage-plugin-notifications-backend-dynamic"
+
+disabled: false
+package: "./dynamic-plugins/dist/backstage-plugin-signals-backend-dynamic"
+----
+
+. (Optional) If using an external database, add the following configuration in your values.yaml:
++
+[source,yaml]
+----
+externalDBsecretRef:
+externalDBName: "<database_name>"
+externalDBHost: "<database_host>"
+externalDBPort: "<database_port>"
```

Comment:
```suggestion
orchestrator:
  sonataflowPlatform:
    externalDBsecretRef: "<cred-secret>"
    externalDBName: "<database_name>"
    externalDBHost: "<database_host>"
    externalDBPort: "<database_port>"
```

Should be under `orchestrator.sonataflowPlatform`: https://github.com/redhat-developer/rhdh-chart/blob/release-1.7/charts/backstage/values.yaml#L378-L388

I think we should also add a link to https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.6/html/configuring_red_hat_developer_hub/configuring-external-postgresql-databases#proc-configuring-postgresql-instance-using-helm_configuring-external-postgresql-databases, as the user would, besides the above, need to update their values.yaml with what is depicted in this procedure.